src/pdf.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  Widget
+} from '@phosphor/widgets';
+
+import {
+  IRenderMime
+} from '@jupyterlab/rendermime-interfaces';
+
+declare var require: any;
+let PDF = require('pdfjs-dist');
+PDF = require('pdfjs-dist/web/pdf_viewer');
```

Comment:
what does doing this duplicate require step do? To be fair given that you declared require immediately above, I can't figure out what require is doing in this case since it wasn't imported…

Reply:
This is kind of messy (followup commits make it a touch less messy), but comes down to some friction between the kind-of old-fashioned PDFJS library and some kind-of inadequate typings. I'd still like to make this a bit cleaner.